src/components/staff-payment/StaffPayment.vue

```diff
@@ -18,7 +18,7 @@
         :sm="displaySideLabel ? 9 : 12"
       >
         <v-radio-group
-          v-model="paymentOption"
+          v-model="staffPaymentData.option"
```

Comment:
Rather than using `paymentOption`, and having to keep it updated with the `staffPaymentData` object, just use `staffPaymentData.option`

Reply:
Pretty slick, not so much here but below where you are able to remove the `:input` event handlers 👍